connectd/peer_exchange_initmsg.c

```diff
@@ -75,12 +75,6 @@ static struct io_plan *peer_init_received(struct io_conn *conn,
 	 *    - MAY fail the connection.
 	 */
 	if (tlvs->networks) {
-		if (!tlvs->networks->chains) {
```

Comment:
nit: this change belongs in a separate commit from the first, as they're not related changes